virttest/libvirt_xml/vm_xml.py

```diff
@@ -3325,3 +3333,24 @@ def __init__(self, virsh_instance=base.virsh):
                                attribute='unit')
         super(VMFeaturesHptXML, self).__init__(virsh_instance=virsh_instance)
         self.xml = '<hpt/>'
+
+
+class VMSysinfoXML(VMXML):
+
+    """
+    sysinfo tag XML class
```

Comment:
Could you paste one simple example how VMSysinfoXML looks like ?

Reply:
Updated